<?xml version="1.0"?>
<launch>
  <param name="robot_description"
    command="$(find xacro)/xacro.py '$(find youbot_description)/robots/youbot_arm.urdf.xacro'" />

  <!-- send fake joint values -->
  <node name="joint_state_publisher" pkg="youbot_kinematics_rviz" type="arm_state_publish" output="screen">
  </node>

  <!-- reconfigure -->
  <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" />

  <!-- Combine joint values -->
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher"/>

  <!-- Show in Rviz   -->
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find youbot_description)/launch/youbot.rviz" />

</launch>
